galata/tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "rootDir": "src",
     "module": "commonjs",
     "types": ["node"],
-    "lib": ["DOM", "DOM.iterable", "es2019.array"]
+    "lib": ["DOM", "DOM.iterable", "es2019.array", "esnext"]
```

Comment:
This seems to be needed according to https://github.com/microsoft/playwright/issues/28262